src/js/home.js

```diff
@@ -174,28 +174,27 @@ const AddForm = {
     },
 
     saveFormToDataBase() {
-        const doc = document.getElementsByClassName('question');
-        let questionList = { name: $id('name').value };
-
-        let number = 1;
+        let formToBase = [];
+        const doc = document.getElementById("addForm-questionList").children;
+        formToBase.push({formName: document.getElementById("addForm-formName").value})
         for (let i = 0; i < doc.length; i++) {
-            if (doc[i].id === 'open question') {
-                questionList[`question${number}`] = doc[i].value;
-            } else {
-                const question = { question: doc[i].value };
-
-                for (let j = i + 1; j < i + 4; j++)
-                    question[`answer${j - 1}`] = doc[j].value;
-
-                i += 3;
-                questionList[`question${number}`] = question;
+            if (doc[i].children[0].id === 'open question')
```

Comment:
Nope, nope, nope
Nie może wystąpić taka sytuacja że kilka elementów ma to samo id, to jest do zmiany